labelit/mainapp/templates/manage_projects/list.html

```diff
@@ -2,6 +2,7 @@
 {% load static %}
 {% block extrahead %}
 <link rel="stylesheet" href="{% static 'table.css' %}">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
this is for delete icon? i think it will be better to use a normal button instead.

Reply:
Removed the icon and kept simple text button.